src/pybind/mgr/dashboard/services/nvmeof_conf.py

```diff
@@ -58,3 +58,21 @@ def remove_gateway(cls, name):
             raise NvmeofGatewayDoesNotExist(name)
         del config['gateways'][name]
         cls._save_config(config)
+    
+    @classmethod
+    def get_client_cert(cls):
+        client_cert = mgr.get_store('nvmeof/mtls_client_cert')
+        if client_cert:
+            return json.dumps(client_cert).encode()
+
+    @classmethod
+    def get_client_key(cls):
+        client_key = mgr.get_store('nvmeof/mtls_client_key')
+        if client_key:
+            return json.dumps(client_key).encode()
+
+    @classmethod
+    def get_server_cert(cls):
+        server_cert = mgr.get_store('nvmeof/mtls_server_cert')
+        if server_cert:
+            return json.dumps(server_cert).encode()
```

Comment:
```suggestion
    @classmethod
    def get_client_cert(cls):
        client_cert = cls.get_store('nvmeof/mtls_client_cert')
        if client_cert:
            return json.dumps(client_cert).encode()

    @classmethod
    def get_client_key(cls):
        client_key = cls.get_store('nvmeof/mtls_client_key')
        if client_key:
            return json.dumps(client_key).encode()

    @classmethod
    def get_server_cert(cls):
        server_cert = cls.get_store('nvmeof/mtls_server_cert')
        if server_cert:
            return json.dumps(server_cert).encode()
```

Reply:
updated

Reply:
i didn't check this earlier when I was updating but cls is the Class object and that doesn't have the `get_store` functionality and this was breaking btw. . I am already importing the mgr so I was actually using that to call the get_store.